qiskit/algorithms/gradients/finite_diff_estimator_gradient.py

```diff
@@ -43,14 +57,27 @@ def __init__(self, estimator: BaseEstimator, epsilon: float, options: Options |
                 The order of priority is: options in ``run`` method > gradient's
                 default options > primitive's default setting.
                 Higher priority setting overrides lower priority setting
+            method: The computation method of the gradients.
+
+                - ``central`` computes :math:`\frac{f(x+e)-f(x-e)}{2e}`,
+                - ``forward`` computes math:`\frac{f(x+e) - f(x)}{e}`,
+                - ``backward`` computes :math:`\frac{f(x)-f(x-e)}{e}`
```

Comment:
Sphinx fails here, maybe there's an indentation missing? 🤔 (I'm no expert on Sphinx though!)
```suggestion
                    - ``central`` computes :math:`\frac{f(x+e)-f(x-e)}{2e}`,
                    - ``forward`` computes math:`\frac{f(x+e) - f(x)}{e}`,
                    - ``backward`` computes :math:`\frac{f(x)-f(x-e)}{e}`
```

Reply:
Thanks